app/javascript/packs/dashboardDropdowns.js

```diff
@@ -0,0 +1,126 @@
+import { initializeDropdown } from '@utilities/dropdownUtils';
+
+function getFormValues(form) {
+  const articleId = form.action.match(/\/(\d+)$/)[1];
+  const inputs = form.getElementsByTagName('input');
+  const formData = { id: articleId, article: {} };
+
+  for (let i = 0; i < inputs.length; i += 1) {
+    const input = inputs[i];
+    const name = input.getAttribute('name');
+    const value = input.getAttribute('value');
+
+    if (name.match(/\[(.*)\]/)) {
```

Comment:
This is something minor but I'd probably move the regex into constants outside the loop instead of re-compiling them on every iteration. This way you can also give them an intent-revealing name.

Reply:
done!